test/run_websocket_server.py

```diff
@@ -116,19 +27,38 @@ def start_websocket_server_worker(
         "--verbose",
         "-v",
         action="store_true",
-        help="if set, websocket server worker will be started in verbose mode",
+        help="""if set, websocket server worker will be started in verbose mode""",
     )
-
+    parser.add_argument(
+        "--notebook",
+        type=str,
+        default="normal",
+        help="""can run websocket server for websockets examples of mnist/mnist-parallel or
+    pen_testing/steal_data_over_sockets. Type 'mnist' for starting server
```

Comment:
Q: Shouldn't there be 1/2 more level of indentation?

Reply:
corrected.